### DISCUSSION TITLE
How to configure API limits in Hasura Cloud?

### DISCUSSION THREAD

**Discussion by jdahlborg** [hasura/graphql-engine#7832] [Question]
Looking at the Hasura Avanced Tutorial, the info seems outdated; https://hasura.io/learn/graphql/hasura-advanced/security/4-rate-limit/
And, it says "Click on Configure to start specifying a rule.". 

Where's that configure button and how do I configure the API limits?

**Comment by zifeo** [comment]
According to https://github.com/hasura/graphql-engine/issues/2151, this feature is only available on the cloud version for now.

**Reply by jdahlborg** [reply]
I'm on the Hasura Cloud but it seems impossible to configure the API limits. It should be possible but I can't figure out how. Any ideas?
